### ISSUE TITLE
Will there be a multilingual version of pretrain model?

### ISSUE THREAD

**Issue by neavo** [AnswerDotAI/ModernBERT#143]

**Comment by jph00** [comment]
There's no current plans for one, although I suspect it might be something you could achieve with a fine-tune.

**Comment by neavo** [comment]
> There's no current plans for one, although I suspect it might be something you could achieve with a fine-tune.目前没有相关计划，不过我怀疑这可能是可以通过微调实现的功能。

After reviewing the data of the tokenizer, it is essentially only Latin script-based text. Therefore, for non-Latin script languages (such as Chinese, Japanese, etc.), continuing pre-training is roughly equivalent to starting training from scratch.
Starting from scratch to pre-train basic language abilities might exceed the capabilities and computational power of most end-users.

**Comment by shahjaidev** [comment]
Agreed, there would be a lot of benefit to the community from a multilingual pre-trained model with a multi-language tokenizer. Simply  finetuning doesn't work

**Comment by jumasheff** [comment]
In a multilingual world you cannot call a model "Modern" if its not designed to support multiple languages.

**Comment by neavo** [comment]
> There's no current plans for one, although I suspect it might be something you could achieve with a fine-tune.

I made some attempts: on the basis of the original model, I first expanded the tokenizer's vocabulary to include information on multilingual (mainly Chinese, Japanese, and Korean) tokens, and then performed secondary pre-training with approximately 1 billion tokens of corpus (5GB of pure text). In terms of results, the model trained did not actually learn the information of non-Latin-based languages.

So, unless there was some mistake in my operation or a bug in the current version of the Hugging Face Transformers library, I believe that mere "fine-tuning" to this extent of secondary training is not sufficient to achieve support for multiple languages. A fully pre-trained official multilingual version is needed.
